XamarinCommunityToolkitSample/ViewModels/Converters/ConvertersGalleryViewModel.cs

```diff
@@ -1,15 +1,19 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using XamarinCommunityToolkitSample.Models.Converters;
+﻿using System.Collections.Generic;
+using Xamarin.Forms;
+using XamarinCommunityToolkitSample.Models;
+using XamarinCommunityToolkitSample.Pages.Converters;
 
 namespace XamarinCommunityToolkitSample.ViewModels.Converters
 {
     public class ConvertersGalleryViewModel : BaseViewModel
     {
-        public IEnumerable<ConverterSectionModel> Items { get; } =
-            ((ConverterSectionId[])Enum
-            .GetValues(typeof(ConverterSectionId)))
-            .Select(id => new ConverterSectionModel(id));
+        public IEnumerable<SectionModel> Items { get; } = new List<SectionModel> {
+            new SectionModel(
+                typeof(ItemTappedEventArgsPage),
+                "ItemTappedEventArgs",
+                Color.FromHex("#498205"),
```

Comment:
@sthewissen What do you think if instead of having those colors hardcoded in the ViewModel, we have kind of a converter, that returns random colors (based on a list of colors pre-defined)? So if tomorrow we add more items to the list we don't have to think about what color to assign.

Reply:
Char0394 @ this is a fantastic idea, Can you share some code for this as I need something like this for my project

Reply:
@maximusfatigue
https://github.com/xamarin/XamarinCommunityToolkit/blob/main/XamarinCommunityToolkit/Views/AvatarView.shared.cs#L204-L206